cmd/kubeadm/app/apis/kubeadm/v1beta3/doc.go

```diff
@@ -19,32 +19,17 @@ limitations under the License.
 // +k8s:deepcopy-gen=package
 // +k8s:conversion-gen=k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm
 
-// Package v1beta1 has been deprecated by v1beta2
-// Package v1beta1 defines the v1beta1 version of the kubeadm configuration file format.
-// This version graduates the configuration format to BETA and is a big step towards GA.
-//
-//A list of changes since v1alpha3:
-//	- "apiServerEndpoint" in InitConfiguration was renamed to "localAPIEndpoint" for better clarity of what the field
-//	represents.
-//	- Common fields in ClusterConfiguration such as "*extraArgs" and "*extraVolumes" for control plane components are now moved
-//	under component structs - i.e. "apiServer", "controllerManager", "scheduler".
-//	- "auditPolicy" was removed from ClusterConfiguration. Please use "extraArgs" in "apiServer" to configure this feature instead.
-//	- "unifiedControlPlaneImage" in ClusterConfiguration was changed to a boolean field called "useHyperKubeImage".
-//	- ClusterConfiguration now has a "dns" field which can be used to select and configure the cluster DNS addon.
-//	- "featureGates" still exists under ClusterConfiguration, but there are no supported feature gates in 1.13.
-//	See the Kubernetes 1.13 changelog for further details.
-//	- Both "localEtcd" and "dns" configurations now support custom image repositories.
-//	- The "controlPlane*"-related fields in JoinConfiguration were refactored into a sub-structure.
-//	- "clusterName" was removed from JoinConfiguration and the name is now fetched from the existing cluster.
+// Package v1beta3 defines the v1beta3 version of the kubeadm configuration file format.
+// This version improves on the v1beta1 format by fixing some minor issues and adding a few new fields.
```

Comment:
I think we should list changes; you can use the same list used in the KEP update

Reply:
> This version improves on the v1beta1

should be v1beta2

i was planning on adding the changes to this list after they merge.